### ISSUE TITLE
Crash on formatting a timezone

### ISSUE THREAD

**Issue by arturdryomov** [mixpanel/mixpanel-android#567]
* Device: BlackBerry BBB100-1
* OS: 8.1.0
* Mixpanel SDK: 5.4.1

```
Fatal Exception: java.lang.AssertionError: No NameTypeIndex match for SHORT_DAYLIGHT
       at android.icu.impl.TimeZoneNamesImpl$ZNames.getNameTypeIndex(TimeZoneNamesImpl.java:724)
       at android.icu.impl.TimeZoneNamesImpl$ZNames.getName(TimeZoneNamesImpl.java:790)
       at android.icu.impl.TimeZoneNamesImpl.getTimeZoneDisplayName(TimeZoneNamesImpl.java:183)
       at android.icu.text.TimeZoneNames.getDisplayName(TimeZoneNames.java:261)
       at java.util.TimeZone.getDisplayName(TimeZone.java:405)
       at java.util.Date.toString(Date.java:1066)
       at org.json.JSONStringer.value(JSONStringer.java:252)
       at org.json.JSONObject.writeTo(JSONObject.java:723)
       at org.json.JSONStringer.value(JSONStringer.java:237)
       at org.json.JSONObject.writeTo(JSONObject.java:723)
       at org.json.JSONObject.toString(JSONObject.java:692)
       at com.mixpanel.android.mpmetrics.AnalyticsMessages$PeopleDescription.toString(AnalyticsMessages.java:204)
       at com.mixpanel.android.mpmetrics.AnalyticsMessages$Worker$AnalyticsMessageHandler.handleMessage(AnalyticsMessages.java:317)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:164)
       at android.os.HandlerThread.run(HandlerThread.java:65)
```

**Comment by dmitry-novikov** [comment]
Hey, MixPanel team, any change this bug to be fixed?

**Comment by dmitry-novikov** [comment]
Can you please apply the same workaround like Facebook did?
https://stackoverflow.com/questions/51453091/android-facebook-sdk-internal-utility-java-lang-assertionerror/52035884#52035884

**Comment by dmitry-novikov** [comment]
@patedit :question:

**Comment by patedit** [comment]
Hey @dmitry-novikov - I will make sure to add this on the next release.

**Comment by patedit** [comment]
I've created a PR here if you want to take a look: https://github.com/mixpanel/mixpanel-android/issues/567

**Comment by patedit** [comment]
Fix has been released in v5.6.7. Thanks for your patience!
